spec/models/concerns/account/finder_concern_spec.rb

```diff
@@ -3,6 +3,37 @@
 require 'rails_helper'
 
 RSpec.describe Account::FinderConcern do
+  describe '.representative' do
+    context 'with an instance actor using an invalid legacy username' do
+      let(:legacy_value) { 'legacy:value' }
```

Comment:
It would make more sense to have something like `localhost:3000` to better illustrate what the legacy value would have been, but functionally it's the same.